lua/lspconfig/java_language_server.lua

```diff
@@ -0,0 +1,20 @@
+local lspconfig = require('lspconfig')
+local configs = require('lspconfig/configs')
+
+local name = 'java_language_server'
+
+configs[name] = {
+	default_config = {
+		cmd = { '/usr/share/java/java-language-server/lang_server_linux.sh'};
```

Comment:
This `cmd` isn't portable.

Reply:
I wasn't aware it has to be, besides where the script is installed will vary. I could add a comment about adding the script to ones PATH and change the cmd to reflect that.

Reply:
If it's not portable, don't include cmd. See jdtls

Reply:
Alright I've removed the cmd and instead added a short description on what to point it to.